src/app/user-register/user-register.component.css

```diff
@@ -24,22 +24,90 @@
     box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.3);
 }
 
+.image-perfil{
+    background-position: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
 .image{
     width: 100%;
     height: 40%;
-    /* background-color: black; */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.img-main{
+    width: 25%;
+    height: 100%;
+    border-radius: 90%;
+    display: flex;
+    align-items: center;
+    padding-top: 50px;
+    font-weight: bold;
+    position: relative;     
+}
+
+#img{
```

Comment:
coloca o que for de # ou seja id no começo e só classe embaixo pra ficar organizado